tests/Functional/PathLocaleNegotiationTest.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+declare(strict_types=1);
+
+namespace tests\Libero\ContentNegotiationBundle\Functional;
+
+use Libero\ContentNegotiationBundle\Exception\NotAcceptableLocale;
+use Symfony\Component\HttpFoundation\Request;
+
+final class PathLocaleNegotiationTest extends FunctionalTestCase
+{
+    /**
+     * @test
+     * @dataProvider anyLocaleProvider
+     */
+    public function it_will_not_negotiate_if_not_configured(?string $header) : void
+    {
+        $kernel = static::getKernel('PathBased');
+
+        $request = Request::create('/');
+        if (null !== $header) {
+            $request->headers->set('Accept-Language', $header);
+        } else {
+            $request->headers->remove('Accept-Language');
+        }
+
+        $response = $kernel->handle($request);
+
+        $this->assertSame(200, $response->getStatusCode());
+        $this->assertSame('und', $response->headers->get('X-Negotiated-Locale'));
+        $this->assertSame('und', $request->getLocale());
+    }
+
+    public function anyLocaleProvider() : iterable
```

Comment:
these values look more specific to `Accept-Language` than generic POSIX locales such as `en_US`